app/models/ability.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+class Ability
+  include CanCan::Ability
+
+  def initialize user
+    user ||= User.new
+
+    if user.admin?
+      admin_abilities
```

Comment:
```suggestion
      admin_abilities user
```
Chưa test lại pk, ẩu Trang

Reply:
dạ e mới test bên user quên chưa test admin